packages/server-wallet/src/db-admin/db-admin.ts

```diff
@@ -24,7 +28,7 @@ export class DBAdmin {
   }
 
   async truncateDB(): Promise<void> {
-    const tables = ['signing_wallets', 'channels', 'nonces'];
+    const tables = [SigningWallet.tableName, Channel.tableName, Nonce.tableName];
```

Comment:
We also need to keep this list in sync with the files in the migrations directory. I think it is already behind those.